katas/langtonant/alfredo/module1.js

```diff
@@ -0,0 +1,140 @@
+export class class1 {
+    constructor() {
+        this.direction = 1 ;
+        this.matriz = new Array(10);
+        this.posi = new Array(2);
+        this.posi[0] = 5;
```

Comment:
As discussed during class, `posi` can be replaced by variables since it does not seem to have a real purpose of being an array